.vscode/launch.json

```diff
@@ -2,21 +2,17 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "type": "node2",
+      "type": "node",
```

Comment:
`node2` is an experimental debugger that still has many known bugs, so we want to default to the stable one (you can always privately switch to `node2` if you prefer living on the cutting edge).